init.zsh

```diff
@@ -93,7 +106,9 @@ function pmodload {
 
   # Load Prezto modules.
   for pmodule in "$pmodules[@]"; do
+    zdebuglog "Started loading %q" $pmodule
```

Comment:
See my detailed comment above about the possibility of avoiding this and instead do automatic debug  instrumentation for _any_ command, scripted or interactive via `TRAPSIG` functions :smiley: